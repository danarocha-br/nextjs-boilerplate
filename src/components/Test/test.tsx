import { render, screen } from '@testing-library/react';

import Test from './';

describe('<Test />', () => {
  it('should render the heading on test component', () => {
    const { container } = render(<Test />);

    expect(screen.getByRole('heading', { name: /test/i })).toBeInTheDocument();

    expect(container.firstChild).toMatchSnapshot();
  });
});
