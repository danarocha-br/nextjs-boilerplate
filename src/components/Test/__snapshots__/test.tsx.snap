// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Test /> should render the heading on test component 1`] = `
<h1>
  Test
</h1>
`;
